### ISSUE TITLE
Update GreyNoise to allow use of new Community API

### ISSUE THREAD

**Issue by bradchiappetta** [Te-k/harpoon#152]
GreyNoise has released a new Community API that is free for any user.

Recommend updating to allow the use of either the Community or Paid API endpoints.

**Comment by bradchiappetta** [comment]
Community API Reference: https://developer.greynoise.io/reference/community-api#get_v3-community-ip

**Comment by christalib** [comment]
Was that made through that PR: https://github.com/Te-k/harpoon/pull/153 ? If so, could we close that issue?

**Comment by bradchiappetta** [comment]
Yes, correct, not sure why it didn't link when the PR was opened, sorry about that!

**Comment by christalib** [comment]
No problems! Thank you for your work!
